geode-core/src/test/java/org/apache/geode/management/internal/cli/shell/GfshHistoryJUnitTest.java

```diff
@@ -92,6 +91,5 @@ public void testClearHistory() throws Exception {
     // clear the history
     gfsh.clearHistory();
     assertEquals(gfsh.getGfshHistory().size(), 0);
-    assertFalse(gfshHistoryFile.exists());
```

Comment:
Suppose that led to its removal... inter webs are suggesting `file.getAbsoluteFile().exists()` on Windows to get better mileage...